api.go

```diff
@@ -235,7 +235,7 @@ func handleAddOrUpdate(keyName string, r *http.Request) (interface{}, int) {
 		// Only if it's NEW
 		switch r.Method {
 		case "POST":
-			keyName = newSession.OrgID + keyName
+			keyName = generateToken(newSession.OrgID, strings.TrimPrefix(keyName, newSession.OrgID))
```

Comment:
how about hiding TrimPrefix-logic inside `generateToken` helper? so calling part won't have to remember to do this